packages/tokens/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/react"],
+  "root": true
+}
```

Comment:
Would you mind to add an override to prevent React imports in this package?

https://eslint.org/docs/latest/rules/no-restricted-imports